shell/platform/windows/BUILD.gn

```diff
@@ -98,6 +98,8 @@ source_set("flutter_windows_source") {
       "win32_window_proc_delegate_manager.cc",
       "win32_window_proc_delegate_manager.h",
     ]
+
+    libs = [ "dwmapi.lib" ]
```

Comment:
Shouldn't this still be in a !winuwp condition?

Reply:
This is in an else block for `if (target_os == "winuwp")` already.